### DISCUSSION TITLE
Foundry Fundamentals lessons is only english cc

### DISCUSSION THREAD

**Discussion by kitezzzGrim** [Cyfrin/foundry-full-course-cu#2881] [Q&A] [answered]
<img width="1499" alt="Snipaste_2024-11-19_19-28-12" src="https://github.com/user-attachments/assets/cad01b4f-3f29-4c91-8064-2d268014a4d9">

I Only have English subtitles, but I want more language options such as Chinese subtitles

**Comment by EngrPips** [comment]
Hello @kitezzzGrim, I show how to get options for other languages in the image below.

![Screenshot 2024-11-19 at 13 32 01](https://github.com/user-attachments/assets/a1d8f9ca-673b-4cf5-a9c9-e63a419c48da)

**Reply by kitezzzGrim** [reply]
wow i know，i find it only section 1 "introduction - foundry Simple Storage" subtitle is only english cc，others have multiple subtitles，maybe you can take a look，thank you
<img width="364" alt="Snipaste_2024-11-20_10-33-48" src="https://github.com/user-attachments/assets/69a7004a-4daf-4e95-a3d8-d8b78c792edf">

**Reply by EngrPips** [reply, marked as answer]
Please open an issue in the `#issue` tab of this GitHub to get the attention of Cyfrin team.
